src/css/main.scss

```diff
@@ -1097,3 +1097,112 @@ div.search-input:focus-within {
   }
 }
 
+// List pages common styling
+.list-page-main {
+  padding: 0 32px;
+  background-color: #F0F1F4;
+
+  .list-page-header {
+    height: 100px;
+
+    .buttons {
+      flex: 1;
+      gap: 8px;
+    }
+
+    .title {
+      font-weight: 500;
+      font-size: 18px;
+      line-height: 22px;
+      color: var(--blue-800);
+    }
+  }
+
+  .list-page-filters {
+    background-color: #FFFFFF;
+    border-radius: 4px;
+    padding: 16px;
+
+    .filter-group {
+      height: 38px;
+    }
+
+    .filters-row {
+      .filter-group > div:first-child {
+        align-items: center;
+
+        label {
+          margin: 0;
+        }
+      }
+    }
+  }
+
+  .action-dropdown-offset {
+    left: 15px !important;
+  }
+
+  .title-text {
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: -0.02em;
+    color: var(--blue-700);
+  }
+
+  .list-page-list-section {
+    background-color: #FFFFFF;
+    border-radius: 4px;
+    margin-top: 32px;
+    margin-bottom: 32px;
+
+    .items-count-circle {
+      width: 30px;
+      height: 30px;
+      border-radius: 50%;
+      border: 2px solid var(--gray-400);
+      display: block;
+    }
+
+
+    .dropdown-menu {
+      box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.15);
+      border-radius: 4px;
+      border: 0 !important;
+
+      a {
+        font-weight: 400;
+        font-size: 15px;
+        line-height: 32px;
+        padding: 0 8px !important;
+      }
+
+      a:hover {
+        background-color: var(--blue-200) !important;
+        border-radius: 4px;
+      }
+    }
+
+    .btn-group {
+      .dropdown-menu {
+        a {
+          font-weight: 400;
+          font-size: 14px !important;
+          line-height: 20px !important;
+          padding: 4px 8px !important;
+
+        }
+      }
+
+      .font-red-ob {
+        color: var(--red-ob) !important;
+
+        &:hover {
+          color: var(--red-ob) !important;
+        }
+      }
+    }
+  }
+}
+
+
```

Comment:
I moved all the styling which were common for purchase order list and invoice list to one common, so we don't have to re-write the same styling for each page